### PR TITLE
add language file for upcoming rewrite of osu!ppy

### PR DESCRIPTION
**Pull request by StrikerFRFX**

### CODE REVIEW

File: src/Presence/osu!ppy.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+  "osu!ppy.viewNews": {
+    "message": "View News",
+    "description": "Displayed for a button to redirect to the news page."
+  },
+
+  "osu!ppy.support": {
+    "message": "Support",
+    "description": "Displayed when viewing the support page."
+  },
```
Comment (EncryptedDev) [inline src/Presence/osu!ppy.json:20]: Add this to general as it can be use universally

File: src/Presence/osu!ppy.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
+{
+  "osu!ppy.viewingNews": {
+    "message": "Viewing All News",
+    "description": "Displayed when viewing all osu! news."
+  },
```
Comment (EncryptedDev) [inline src/Presence/osu!ppy.json:5]:
```suggestion
  "osu!ppy.viewingNews": {
    "message": "Viewing News Articles",
    "description": "Displayed when viewing all osu! news."
  },
```

File: src/Presence/osu!ppy.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+  "osu!ppy.readingNews": {
+    "message": "Reading News:",
+    "description": "Displayed when reading a news page."
+  },
+
+  "osu!ppy.viewNews": {
+    "message": "View News",
+    "description": "Displayed for a button to redirect to the news page."
+  },
```
Comment (EncryptedDev) [inline src/Presence/osu!ppy.json:15]:
```suggestion
  "osu!ppy.viewNews": {
    "message": "View News Article",
    "description": "Displayed for a button to redirect to the news page."
  },
```

File: src/Presence/osu!ppy.json [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+  "osu!ppy.support": {
+    "message": "Support",
+    "description": "Displayed when viewing the support page."
+  },
+
+  "osu!ppy.beatMapListing": {
+    "message": "Beatmap Listing",
+    "description": "Displayed when viewing all beatmaps."
+  },
```
Comment (EncryptedDev) [inline src/Presence/osu!ppy.json:25]:
```suggestion
  "osu!ppy.beatMapListing": {
    "message": "Viewing Beatmap Listings",
    "description": "Displayed when viewing all beatmaps."
  },
```

File: src/Presence/osu!ppy.json [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+  "osu!ppy.beatMapPacks": {
+    "message": "Beatmap Packs",
+    "description": "Displayed when viewing all beatmap packs."
+  },
```
Comment (EncryptedDev) [inline src/Presence/osu!ppy.json:30]:
```suggestion
  "osu!ppy.beatMapPacks": {
    "message": "Viewing Beatmap Packs",
    "description": "Displayed when viewing all beatmap packs."
  },
```

File: src/Presence/osu!ppy.json [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+  "osu!ppy.beatMapLooking": {
+    "message": "Looking at Beatmap:",
+    "description": "Displayed when looking at a beatmap."
+  },
```
Comment (EncryptedDev) [inline src/Presence/osu!ppy.json:35]:
```suggestion
  "osu!ppy.beatMapLooking": {
    "message": "Looking at Beatmap",
    "description": "Displayed when looking at a beatmap."
  },
```

File: src/Presence/osu!ppy.json [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+  "osu!ppy.countryRankings": {
+    "message": "Country Rankings {0}",
+    "description": "Example: Country Rankings [osu!]"
+  },
+
+  "osu!ppy.beatmapRankings": {
+    "message": "Kudosu Rankings",
+    "description": "Displayed when viewing Kudosu Rankings."
+  },
```
Comment (EncryptedDev) [inline src/Presence/osu!ppy.json:65]:
```suggestion
  "osu!ppy.beatmapRankings": {
    "message": "Viewing Kudosu Rankings",
    "description": "Displayed when viewing Kudosu Rankings."
  },
```

File: src/Presence/osu!ppy.json [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+  "osu!ppy.beatmapView": {
+    "message": "View Beatmap",
+    "description": "Displayed for a button to redirect to the beatmap page."
+  },
+
+  "osu!ppy.wikiMainPage": {
+    "message": "Wiki Main Page",
+    "description": "Displayed when viewing the wiki main page."
+  }
```
Comment (EncryptedDev) [inline src/Presence/osu!ppy.json:75]: This should be "View Wiki" and moved to general

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: src/Presence/osu!ppy.json (hunk 8 above)
Comment (StrikerFRFX) [reply src/Presence/osu!ppy.json:75]: Not used for a button, would be "Viewing Wiki"

File: src/Presence/osu!ppy.json (hunk 1 above)
Comment (EncryptedDev) [reply src/Presence/osu!ppy.json:20]: Not resolved

File: src/Presence/osu!ppy.json (hunk 8 above)
Comment (EncryptedDev) [reply src/Presence/osu!ppy.json:75]: OK, you should still add a verb to it

Comment (StrikerFRFX) [reply src/Presence/osu!ppy.json:75]: It counts as the same as Support so no verb

**Comment by EncryptedDev** [comment]
Closed until PreMiD/Presences PR is created

**Comment by StrikerFRFX** [comment]
Reopen

**Comment by theusaf** [comment]
https://github.com/PreMiD/Presences/pull/6893

**Comment by DarkVillager** [comment]
#7285

**Pull request closed without merging**